FileManager/src/org/openintents/filemanager/search/SearchSuggestionsProvider.java

```diff
@@ -1,6 +1,7 @@
 package org.openintents.filemanager.search;
 
 import android.app.SearchManager;
+import android.bluetooth.BluetoothClass;
```

Comment:
This file is not used.

Reply:
fixed it on https://github.com/openintents/filemanager/pull/130/commits/809b96352871a5e382b1f2db7b1c5fe3e0dc54ee